### PR TITLE
Don't close on select cloud build options

### PR DESCRIPTION
**Pull request by McGiverGim**
This PR modifies the behaviour of the options select from cloud build. Now it remains open, after each selection making it easier for the user to select/unselect more than one option.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
